current/link-analysis.md

```diff
@@ -168,7 +198,7 @@ RecordLoader.loadArchives("example.arc.gz", sc).keepValidPages()
   .filter(r => r._2 != "" && r._3 != "")
   .countItems()
   .filter(r => r._2 > 5)
-  .saveAsTextFile("sitelinks-by-date/")
+  .saveAsTextFile("sitelinks-by-date-rdd/")
 ```
 
 The format of this output is:
```

Comment:
question: line 205 "- Field one: Crawldate" should it be `yyyyMMdd` or `yyyymmdd`?

Reply:
Also line 220, ExtractLinks --> `ExtractLinks` ?